self-service-portal/src/components/common/ParticipantInfo.tsx

```diff
@@ -20,7 +20,9 @@ const ParticipantInfo = () => {
   const authToken = useSelector((state: RootState) => state.tokenReducer.participantToken);
   console.log("part details in dash", participantDetails, authToken);
 
+  useEffect(()=> {
   if(authToken == "abcd"){
+    console.log("came in abcd",)
```

Comment:
What is this logic?

Reply:
This is to see if actual token is generated and stored or not .. by default it is abcd to differentiate between actual token and dummy token